Cargo.toml

```
@@ -11,7 +11,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 rayon = "1.5"
-numpy = "0.16.2"
+numpy = "0.17.0"
```

Is there a reason to use `0.17.0` here instead of `0.17.2` like the other bump?

---

not particularly, the cargo.lock has 0.17.2 now as that gets pulled in by the this, but it probably makes sense to update this in lockstep to ensure we're getting all the bugfixes as the required version (not just relying on our lock version)

```suggestion
numpy = "0.17.2"
```